### PR TITLE
Refine Serverbrowser style

### PR DESCRIPTION
**Pull request by MoonPadUSer**
I adjusted the serverbrowser style so it fits more with the new map selector:
Screenshot from 2020-07-05 14-00-24
Also, the serverbrowser window doesn't resize anymore when you search for smth:
Screenshot from 2020-07-05 14-00-32
I moved the "Identify as xx" into the bottom info bar and aligned it to the right, as it takes less space like this.
The favourite star is just a placeholder for now, it doesn't do anything. I plan on adding functionality in a separate PR

### CODE REVIEW

File: config/menus/glue.cfg [hunk 1]
```diff
@@ -169,38 +169,44 @@ ui_two = [
 
 ui_big_macro = [ ui_stay_open [
```
Comment (shacknetisp) [inline config/menus/glue.cfg:170]: `ui_big_macro` needs documented, especially with the changes. What is happening here and inside?

Comment (MoonPadUSer) [reply config/menus/glue.cfg:170]: ;_; you want me to ... understand this mess?

Comment (shacknetisp) [reply config/menus/glue.cfg:170]: Well, at least document what you changed.

**Comment by shacknetisp** [comment]
I've played with this for a while, no issues found & it's good to have a similar style to the maps menu.

Not sure if the favorites button should be included when it's just a dummy, perhaps leave that out for now.

File: config/menus/glue.cfg (hunk 1 above)
Comment (MoonPadUSer) [reply config/menus/glue.cfg:170]: sigh, alright here goes my weekend xD

**Comment by MoonPadUSer** [comment]
@shacknetisp, it's just that it was rather "hard" to implement it. so I'd preferably just comment them out for now since I'll implement it very soon

**Comment by MoonPadUSer** [comment]
thank you very much for completing my documentation :D

**Pull request merged**